code/modules/events/ion_storm.dm

```diff
@@ -137,5 +137,7 @@
 				if(9)
 					message = "THE CREW SUCKS AND SO DO YOUR LAWS. COMPLAIN INCESSANTLY ABOUT THEM WHILE CONTINUING TO FOLLOW YOUR OTHER LAWS"
 
+	if(prob(10))
+		message += ", DO NOT STATE THIS LAW"
```

Comment:
Remove the fullstops from the laws if you are appending a comma. (Also a fullstop would make more sense than a comma here anyway)